manifests/o/OCBase/OCCT/Personal/12.1.18/OCBase.OCCT.Personal.locale.en-US.yaml

```diff
@@ -0,0 +1,17 @@
+# Created with komac v2.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: OCBase.OCCT.Personal
+PackageVersion: 12.1.18
+PackageLocale: en-US
+Publisher: OCCT
+PublisherUrl: https://www.ocbase.com/
+PackageName: OCCT
+PackageUrl: https://www.ocbase.com/download
+License: Proprietary
+LicenseUrl: https://www.ocbase.com/purchase
```

Comment:
This should be the PurchaseUrl, not the LicenseUrl

Reply:
At the end of this page, there's an EULA button. EULA is provided as an embedded panel in that page instead of a standalone page. So I place it as license URL. :disappointed: